python/idsse_common/idsse/common/sci/grid_proj.py

```diff
@@ -49,12 +50,12 @@ class GridProj:
     """
     def __init__(self,
                  crs: CRS,
-                 lower_left_lat: Optional[float],
-                 lower_left_lon: Optional[float],
+                 lower_left_lat: float | None,
+                 lower_left_lon: float | None,
                  width: float,
                  height: float,
                  dx: float,
-                 dy: Optional[float] = None):
+                 dy: float | None = None):
```

Comment:
In a previous file you used `dy: float, optional = None`

Reply:
Good catch. I think the "float, optional" has showed up when I allowed VSCode to auto-generate the docstring, vs. "float | None" is what I typed manually. I'll change this to use "float, optional"

Reply:
Oh, figured it out. "optional" means a default value is defined for the argument, vs. "float | None" just means the value can be None. I'll fix these type hints now.